src/librustc_codegen_ironox/Cargo.toml

```diff
@@ -11,3 +11,4 @@ test = false
 
 [dependencies]
 ar = "0.6.0"
+goblin = "0.0.18"
```

Comment:
I would have used `0.0` to get the newest API-compatible version available. What do you think? Same for `ar`?

Reply:
I tend to agree (there again, i've not seen `0.0.X` as a version number before, so it might be that it's *highly* unstable and that specifying `0.0.18` is the only safe thing to do).

Reply:
I've seen it on a couple of crates. I don't know if it has any special meaning.